js/main.js

```diff
@@ -1 +1,54 @@
-/* Javascript goes here! */
+$(function() {
+
+  GetWeather();
+
+  $("form").submit(function(e) {
+    e.preventDefault();
+    GetWeather();
+  });
+
+});
+
+function GetWeather() {

```

Comment:
This works, but standard naming convention for javascript is to use camelcase for functions where the first letter is lower-case.